tasks/discover-granules/tests/index.js

```diff
@@ -53,7 +71,7 @@ test('discover granules sets the correct dataType for granules', async (t) => {
 test('discover granules using FTP', async (t) => {
   const { event } = t.context;
 
-  event.config.collection.provider_path = '/granules/fake_granules';
+  event.config.collection.provider_path = 'granules/^fake_granules$';
```

Comment:
This had to change because I added a new directory to the test-data (granules/disco_granules_duplicate_fake_granules).    I spent an hour I won't get back tracking this down in code before just reading the docs, but each *segment* is a regexp, so /granules/fake_granules actually finds `*fake_granules*` in `/granules/*`

This makes the path *explicit* in that it's looking only for fake_granules.